.oscal/ssp.oscal.yaml

```
@@ -0,0 +1,162 @@
+system-security-plan:
+  uuid: 8c1ee2a6-bd14-4f77-b2f7-3d2bc0aaa55e
+  metadata:
+    title: "Oscal Server Software Development System Security Plan"
+    last-modified: "2024-10-24T00:00:00Z"
+    version: "1.0"
+    oscal-version: "1.0.4"
```

Question: why 1.0.4 and not current stable (1.1.2)?

```suggestion
    oscal-version: "1.1.2"
```

---

fixed